Update Heartbeat check overview page

guolau opened the pull request:
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [x] Code is linted (`npm run lint`)

## Notes for the Reviewer
Updated the Heartbeat check overview page according to the [check overview page facelift proposal](https://www.notion.so/checkly/Checkly-Docs-Overview-Pages-Proposal-18bec050b06e8091b408e2b47035c3b6).

ragog commented on site/content/docs/heartbeat-checks/_index.md:
````diff
@@ -14,159 +14,136 @@ aliases:
+wget -T 5 -t 3 https://ping.checklyhq.com/87c05896-3b7d-49ae-83ff-5e81323a54c4
+```
+
+You can use curl in the [Heroku Scheduler](https://devcenter.heroku.com/articles/scheduler):
````
@guolau wdyt about exchanging this with Vercel cron jobs?

guolau replied:
Removing the Heroku example and adding a Vercel one? I think that'd be good, Vercel seems more popular atm.

Vercel has examples for different JS frameworks. I could create a separate section for these examples, and copy and modify their examples for heartbeat pings.

![image](https://github.com/user-attachments/assets/b86bb74f-c1e8-454f-b2ff-ddefa4ced263)

ragog reviewed (changes requested):
@guolau looking good, let's address some minor points and then we're good to go.

Dgiordano33 commented:
@guolau @ragog - this looks great, love the change to Vercel too from Heroku, makes way more sense. Two quick things:

1. This wasn't changed, so just feedback on the OG doc - I think the Video and Image are both too high at the top, and distract from when you land on the page. Think we could move the "What is Heartbeat Checks" text to above the nice intro image. Or just have the video first, then text, then maybe more text, then the image. 

2. Again, feedback on the OG doc - this is a single page and it's so dense. Can any of this be moved to another page in the Heartbeat Section dropdown?

guolau commented:
@Dgiordano33 Thanks for looking over this!
1. I see your point here, let me play around with this. The other docs pages ([API](https://www.checklyhq.com/docs/api-checks/), [Browser](https://www.checklyhq.com/docs/browser-checks/) have a similar format, so I'll make a note to adjust those too (at some point).
2. Let me make the Ping examples its own page, there are quite a few of those 👀

guolau commented on site/content/docs/heartbeat-checks/_index.md:
```diff
@@ -16,19 +16,15 @@ aliases:
 
-![Heartbeat check overview page](/docs/images/heartbeat-checks/heartbeat-check-overview.png)
-
-Check out this video for a quick explainer:
```
I removed this video. It takes up a lot of space and there's not really a good place for it.

It would be nice to incorporate the YouTube videos more into the docs, though. We have useful videos about all the check types, we just need a more compact format. Something like this, but for videos:
![image](https://github.com/user-attachments/assets/981dfbf1-b945-4c92-baa2-27c3e9f6a520)

guolau commented:
Summary of changes:
* Moved the first image further down the page
* Removed embedded video
* Ping examples
   * Removed Heroku as a separate example
   * Added Vercel cron job examples
   * Made Ping examples its own docs page

ragog commented:
@Dgiordano33 want to give your final OK on this?

Dgiordano33 commented:
Looks great @guolau - much more balanced and super actionable!

ragog reviewed (approved):
LGTM @guolau
